google/cloud/security/common/gcp_api/bigquery.py

```diff
@@ -59,19 +59,22 @@ def get_bigquery_projectids(self):
             If there are no project_ids enabled for bigquery an empty list will
             be returned.
         """
+        key='projects'
```

Comment:
Hmmmm.  I feel that this should not be passed in as an arg for a few reasons:
- the method naming defines a very specific data is retrieved
- the response data structure is very rigid
- even if the the response data structure changes, this should be fixed inside the api method here, rather than letting the caller changing the key